xml/System.Windows.Forms/ToolStripControlHost+ToolStripHostedControlAccessibleObject.xml

```diff
@@ -34,10 +34,9 @@
         <Parameter Name="toolStripControlHost" Type="System.Windows.Forms.ToolStripControlHost" />
       </Parameters>
       <Docs>
-        <param name="toolStripHostedControl">To be added.</param>
-        <param name="toolStripControlHost">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="toolStripHostedControl">The ToolStrip control hosted in the ToolStripControlHost container.</param>
+        <param name="toolStripControlHost">The ToolStripControlHost container which hosts the control.</param>
+        <summary> Creates the new instance of ToolStripHostedControlAccessibleObject. </summary>
```

Comment:
A constructor can have a cref to describe what you're initializing. To disambiguate from other constructors (if there are any), you can also describe what is the purpose of the passed parameters:

```suggestion
        <summary> Initializes a new <see cref="T:System.Windows.Forms.ToolStripControlHost.ToolStripHostedControlAccessibleObject" /> instance with the specified tool strip hosted control and tool strip control host. </summary>
```